tests/unit/core/data/transform_libs/test_mmseg.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Helper to support MMPretrain data transform functions."""
+from __future__ import annotations
+
+from unittest.mock import MagicMock
+
+import numpy as np
+import pytest
+import torch
+from otx.core.data.entity.base import ImageInfo
+from otx.core.data.entity.segmentation import SegDataEntity
+from otx.core.data.transform_libs.mmseg import LoadAnnotations, MMSegPackInputs, PackSegInputs
+
+
+@pytest.fixture()
+def data_entity() -> tuple[tuple, SegDataEntity]:
+    img_size = (224, 224)
+    data_entity = SegDataEntity(torch.rand(img_size),
+                                ImageInfo(0, img_size[0], img_size[0],
+                                          img_size[0], img_size[0]),
+                                torch.rand(img_size))
+    return img_size, data_entity
```

Comment:
There seems duplication for this fixture. You can put this into more common place. For example,

1. Fixture definition
```python
# tests/unit/core/conftest.py

@pytest.fixture()
def fxt_seg_data_entity() -> tuple[tuple, SegDataEntity]:
    img_size = (224, 224)
    data_entity = SegDataEntity(torch.rand(img_size),
                                ImageInfo(0, img_size[0], img_size[0],
                                          img_size[0], img_size[0]),
                                torch.rand(img_size))
    return img_size, data_entity
```

2. Fixture usage
```python
# tests/unit/core/data/transform_libs/test_mmseg.py

class TestLoadAnnotations:
    @pytest.fixture()
    def annotations_loader(self) -> LoadAnnotations:
        return LoadAnnotations()

    def test_transform(self, fxt_seg_data_entity, annotations_loader) -> None:
```